### DISCUSSION TITLE
hi! i have a question.

### DISCUSSION THREAD

**Discussion by macubukujkun** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#334] [Q&A] [answered]
hello, i'm korean. (south of korea)
i don't know English very well

i was download the Aldente app.
setting charge limit : 80%
but Mac OS show me, 83%... why??

maybe...
This is processing to protect the battery, right?
(for example : Mac OS 100%, Real battery charging 97%)

i have a question.
if i want to unstall aldente,  should I set it up 97% before deleting it? Or do I have to do it 100%?

If I set it 100% and delete it, MacOS would be 103%
also real battery (100%) is dangerous.
(because over charging)

i use Macbook Pro 2015, 15"
i use Altende free app

thanks.

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @macubukujkun,

The charge limiter is a little bit inaccurate on old (pre-2016) MacBooks. This is due to AlDente using the "real" battery percentage whereas macOS is using a different battery percentage. However, you don't have to worry about damaging your battery by setting the charge limit to 100% with AlDente. MacOS will still keep a safety margin for your battery. Therefore, when before uninstalling AlDente set the charge limiter back to 100%.

**Reply by macubukujkun** [reply]
thanks :)
